### PR TITLE
Todo list add

### PR DESCRIPTION
**Pull request by faisalakb**
Please check my Todo list PR that has the following features: 💯
- Create an array of objects. 👍
- use local storage. 👍
- add dynamic HTML. 👍
- Handle Folder and file structure using WebPack. 👍

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import './style.css';
+
+const todoList = [];
+let list = [];
```
Comment (Davidosky007) [inline src/index.js:3]: * According to the requirements you are expected to 
* Create an index.js file and set an array of some simple to-do tasks (array of objects).
* Write a function to iterate over the tasks array and populate an HTML list item element for each task
* On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
* Kindly make the necessary adjustments 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @faisalakb 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* ES6 syntax used properly 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @faisalakb 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Listing the todo list items dynamically ✔️
- Correct use of use functional component that iterates over the array of objects ✔️
- Well-structured files :heavy_check_mark: 


Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Comment by faisalakb** [comment]
Thanks for Review

**Pull request merged**